File: searcher/src/main/java/searcher/model/Lessee.java
package searcher.model;

import javafx.beans.property.*;

public class Lessee {
    
        //Declare Lessee Table Columns
        private IntegerProperty lessee_id;
        private IntegerProperty unit_id;
        private StringProperty unitLabel;
        private StringProperty lesseename;
        private StringProperty addrl1;
        private StringProperty addrl2;
        private StringProperty city;
        private StringProperty state;
        private IntegerProperty zip;
        private StringProperty phone;
        private BooleanProperty active;
    
        //Constructor
        public Lessee() {
            this.lessee_id = new SimpleIntegerProperty();
            this.unit_id = new SimpleIntegerProperty();
            this.unitLabel = new SimpleStringProperty();
            this.lesseename = new SimpleStringProperty();
            this.addrl1 = new SimpleStringProperty();
            this.addrl2 = new SimpleStringProperty();
            this.city = new SimpleStringProperty();
            this.state = new SimpleStringProperty();
            this.zip = new SimpleIntegerProperty();
            this.phone = new SimpleStringProperty();
            this.active = new SimpleBooleanProperty();
        }
    
        //lesseeid
        public int getLesseeId() {
            return lessee_id.get();
        }
        
        public void setLesseeId(int lesseeId){
            this.lessee_id.set(lesseeId);
        }
        
        public IntegerProperty lesseeIdProperty(){
            return lessee_id;
        }
    
        //unitid
        public int getUnitId() {
            return unit_id.get();
        }
        
        public void setUnitId(int unitId){
            this.unit_id.set(unitId);
        }
        
        public IntegerProperty unitIdProperty(){
            return unit_id;
        }

        //unitLabel
        public String getUnitLabel() {
            return unitLabel.get();
        }
        
        public void setUnitLabel(String unitLabel){
            this.unitLabel.set(unitLabel);
        }
        
        public StringProperty unitLabelProperty(){
            return unitLabel;
        }
    
        //lesseename
        public String getLesseeName() {
            return lesseename.get();
        }
        
        public void setLesseeName(String lesseeName){
            this.lesseename.set(lesseeName);
        }
        
        public StringProperty lesseeNameProperty(){
            return lesseename;
        }
    
        //addrl1
        public String getaddrL1() {
            return addrl1.get();
        }
        
        public void setAddrL1(String AddrL1){
            this.addrl1.set(AddrL1);
        }
        
        public StringProperty addrL1Property(){
            return addrl1;
        }
    
        //addrl2
        public String getaddrL2() {
            return addrl2.get();
        }
        
        public void setAddrL2(String AddrL2){
            this.addrl2.set(AddrL2);
        }
        
        public StringProperty addrL2Property(){
            return addrl2;
        }
    
        //city
        public String getCity() {
            return city.get();
        }
        
        public void setCity(String City){
            this.city.set(City);
        }
        
        public StringProperty cityProperty(){
            return city;
        }
    
        //state
        public String getState() {
            return state.get();
        }
        
        public void setState(String State){
            this.state.set(State);
        }
        
        public StringProperty stateProperty(){
            return state;
        }
    
        //zip
        public int getZip() {
            return zip.get();
        }
        
        public void setZip(Integer Zip){
            this.zip.set(Zip);
        }
        
        public IntegerProperty zipProperty(){
            return zip;
        }
    
        //phone
        public String getPhone() {
            return phone.get();
        }
        
        public void setPhone(String Phone){
            this.phone.set(Phone);
        }
        
        public StringProperty phoneProperty(){
            return phone;
        }

        //active
        public Boolean getActive() {
            return active.get();
        }
        
        public void setActive(Boolean Active){
            this.active.set(Active);
        }
        
        public BooleanProperty activeProperty(){
            return active;
        }
    }

File: python/searcher/Lessee.py
class Lessee:
    def __init__(self, lessee_id, unit_id, unit_label, lessee_name, addrL1, addrL2, city, state, zip, phone, active):
        self._lesseeId = int(lessee_id)
        self._unitId = int(unit_id)
        self._unitLabel = str(unit_label)
        self._lesseeName = str(lessee_name)
        self._addrL1 = str(addrL1)
        self._addrL2 = str(addrL2)
        self._city = str(city)
        self._state = str(state)
        self._zip = int(zip)
        self._phone = str(phone)
        self._active = bool(active)

    def get_lesseeId(self):
        return self._lesseeId
    
    def set_lesseeId(self, value):
        self._lesseeId = value

    def get_unitId(self):
        return self._unitId
    
    def set_unitId(self, value):
        self._unitId = value

    def get_unitLabel(self):
        return self._unitLabel
    
    def set_unitLabel(self, value):
        self._unitLabel = value

    def get_lesseeName(self):
        return self._lesseeName
    
    def set_lesseeName(self, value):
        self._lesseeName = value

    def get_addrL1(self):
        return self._addrL1
    
    def set_addrL1(self, value):
        self._addrL1 = value

    def get_addrL2(self):
        return self._addrL2
    
    def set_addrL2(self, value):
        self._addrL2 = value

    def get_addrL1(self):
        return self._addrL1
    
    def set_addrL1(self, value):
        self._addrL1 = value

    def get_city(self):
        return self._city
    
    def set_city(self, value):
        self._city = value

    def get_state(self):
        return self._state
    
    def set_state(self, value):
        self._state = value

    def get_zip(self):
        return self._zip
    
    def set_zip(self, value):
        self._zip = value    
        
    def get_phone(self):
        return self._phone
    
    def set_phone(self, value):
        self._phone = value

    def get_active(self):
        return self._active
    
    def set_active(self, value):
        self._active = value